nuxeo-importer-queues/src/main/java/org/nuxeo/ecm/platform/importer/queue/manager/CQManager.java

```diff
@@ -1,4 +1,5 @@
-package org.nuxeo.ecm.platform.importer.queue.manager;/*
+package org.nuxeo.ecm.platform.importer.queue.manager;
```

Comment:
Please put the `package` declaration after the license header (which should be Apache btw).